platform/security/security-oidc-realm/src/main/resources/OSGI-INF/metatype/metatype.xml

```diff
@@ -21,7 +21,7 @@
 
         <AD name="OIDC NameID Policy" id="usernameAttributeList"
             type="String" cardinality="100"
-            default="name, email"
+            default="preferred_username, name, email, sub"
```

Comment:
📘 Updated to reflect OIDC defaults

Reply:
❓ Having the type be String here still works? As in if we change this value in the metatype and check the username attribute it sets it as a list and not one string with all the values?

Reply:
Correct. I checked metatypes other than the saml one and thats how they do it